src/configuration/ArcanistSettings.php

```diff
@@ -188,6 +188,19 @@ private function getOptions() {
         'default' => false,
         'example' => 'false',
       ),
+      'uber.land.submitqueue.enable' => array(
```

Comment:
IMO it would be enough to have `uber.land.submitqueue.uri` config only. If value is `null` or `false` - submit queue is disabled.